lighthouse-core/gather/gatherers/form-elements.js

```diff
@@ -40,15 +40,16 @@ function collectFormElements() {
           name: parentFormElement.name,
           autocomplete: parentFormElement.autocomplete,
           // @ts-expect-error - getNodeDetails put into scope via stringification
-          ...getNodeDetails(parentFormElement),
+          node: getNodeDetails(parentFormElement),
```

Comment:
since this is now a thing, i think it makes more sense as a sibling to `artifacts`.